### DISCUSSION TITLE
Feedback: Document example usage of plugins

### DISCUSSION THREAD

**Discussion by alexcroox** [nuxt/eslint-config#372] [Ideas]
I thought I'd give some feedback that might help others in your docs. This is the first I've heard of this new upcoming flat eslint config. One thing I can't work out when I look to migrate my existing .eslintrc.cjs file is how to utilise plugins and "extends".

e.g

```
extends: [
    'plugin:drizzle/recommended',
    'plugin:vitest/recommended'
  ],

  plugins: ['simple-import-sort', 'import', 'drizzle', 'vitest'],
```

An example showing a popular plugin e.g Drizzle, and where to put the `plugin:drizzle/recommended` within the `withNuxt` object would be very helpful.

Thank you!

**Comment by alexcroox** [comment]
I believe the eslint docs are here, assuming they translate to `withNuxt` usage/layout https://eslint.org/docs/latest/use/configure/migration-guide#predefined-and-shareable-configs

**Comment by m-shum** [comment]
Not gonna lie, the documentation of this module is quite confusing.

The 'manual' setup has an eslint object in the nuxt config _and_ an eslint config file. Are you supposed to copy all of your options across both? Because that seems redundant. 

A working example of this setup would be great because it is quite different to the way things used to work in Nuxt 2.
